### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by bkBrunobk**
v0.24.0 translations tuned

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -67,7 +67,7 @@
     <string name="picture_url">Url de l\'image</string>
     <string name="description">Description</string>
     <string name="about_us">"à propos.. "</string>
-    <string name="what_s_on_your_mind">Qu\'es ce qui vous préoccupe ?</string>
+    <string name="what_s_on_your_mind">Quoi de neuf ?</string>
```
[the remark this answers is not in the document]
Comment (bkBrunobk) [reply app/src/main/res/values-fr/strings.xml:70]: Hello @Chemaclass !
You propose to suppress the 'space' for obtaining a shorter string ?

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -67,7 +67,7 @@
     <string name="picture_url">Url de l\'image</string>
     <string name="description">Description</string>
     <string name="about_us">"à propos.. "</string>
```
[the remark this answers is not in the document]
Comment (bkBrunobk) [reply app/src/main/res/values-fr/strings.xml:69]: I don't know where this string is displayed.
But your suggestion seems better.

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bkBrunobk) [reply app/src/main/res/values-fr/strings.xml:70]: Ther was'nt any space at the end of the sentence but a '?'
Quoi de neuf **?**
Spanish : qué hay de nuevo ?

[the remark this answers is not in the document]
Comment (Chemaclass) [reply app/src/main/res/values-fr/strings.xml:70]: In [Spanish](https://github.com/vitorpamplona/amethyst/blob/main/app/src/main/res/values-es/strings.xml#L70): `res/values-es/strings.xml`
```xml
<string name="what_s_on_your_mind">¿Qué tienes en mente?</string>
```

[the remark this answers is not in the document]
Comment (bkBrunobk) [reply app/src/main/res/values-fr/strings.xml:70]: In french we always put a space before the word and '?'
¿Qué tienes en mente? --FR--> à quoi penses tu ?

"Quoi de neuf ?" was taken from Twitter (probably a bad idea)

[the remark this answers is not in the document]
Comment (Chemaclass) [reply app/src/main/res/values-fr/strings.xml:70]: Oh! I didn't know about this rule: https://www.englishforums.com/English/SpaceBeforeOr/kkqlm/post.htm 
In such a case, you can/must leave that empty space before the question mark 👍🏼

**Pull request merged**